arch/arm/src/cxd56xx/cxd56_sfc.c

```diff
@@ -82,7 +82,8 @@ static int cxd56_erase(FAR struct mtd_dev_s *dev, off_t startblock,
   int ret;
   size_t i;
 
-  finfo("erase: %08lx (%u blocks)\n", startblock << PAGE_SHIFT, nblocks);
+  finfo("erase: %" PRIxOFF " (%u blocks)\n",
```

Comment:
Besides the change to the format specifier, the 8 leading zeroes also have been removed.
I am not raising an issue, just to confirm whether this was intentional or by mistake.

Reply:
Yes, it's my intention to remove 08 since startblock isn't always 4bytes after this patch which make the prefix isn't right in 64bit case.